### DISCUSSION TITLE
Mute Animation

### DISCUSSION THREAD

**Discussion by nate-26** [end-4/dots-hyprland#175] [Q&A] [answered]
I was wondering if there was a way to configure an indicator for when I toggle mute? I use this system in class so, being very paranoid, I'd like to be able to see when I actually toggle (and untoggle) mute. Forgive me if I'm missing something, the mute keybind is working fine, I just can't tell it's muted until I pull up the audio settings or if I am playing something with audio. I checked the keybinds.conf as well as the ags indicator .js files and there doesn't seem to be an animation that I broke.

**Comment by end-4** [comment]
hmm yes added this to my todo list

**Comment by end-4** [comment, marked as answer]
added https://github.com/end-4/dots-hyprland/commit/d8df1eef05fdb6954c4a8580d752e3711ca1df95
though the mute keybind is no longer real mute. this sets volume to 0 as i believe this is more user-friendly and is safer (imagine leaving at 100% then unmute unexpectedly) ~~and it's easier to code as i'm lazy~~

**Reply by nate-26** [reply]
Thank you so much!
